client/core/core.go

```diff
@@ -4547,6 +4568,16 @@ func (c *Core) initializeDEXConnections(crypter encrypt.Crypter) []*DEXBrief {
 			continue
 		}
 
+		// If the connection is down, authDEX will fail on Send.
```

Comment:
We were already sending a notification when authDEX failed. Is this about setting the connection icon on the front end?

Reply:
This is mainly about avoiding a confusing error from authDEX about a broken connection or something when we know it won't be possible.  Nothing goes terribly wrong without these lines, but it's a smoother UX. (`TopicDEXDisconnected` with host name vs. `TopicDexAuthError` with an error message from the comms methods)